packages/block-editor/src/components/block-tools/use-block-toolbar-popover-props.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+const TOOLBAR_HEIGHT = 72;
```

Comment:
We should try not to store hardcoded values like this one:

- we could either read it from the toolbar (probably the easiest alternative)
- or we could make it so that the value is originally stored in JS, and they applied to the toolbar (maybe via custom props)?

Reply:
Definitely. I've added some code that uses the popover's `offsetHeight`. 👍